app/src/main/res/layout/movie_item.xml

```diff
@@ -1,52 +1,50 @@
 <?xml version="1.0" encoding="utf-8"?>
-<com.google.android.material.card.MaterialCardView
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:clickable="true"
-    android:focusable="true"
-    android:foreground="@drawable/ripple"
-    app:cardCornerRadius="@dimen/movie_radius_8"
-    app:strokeColor="@color/border_color"
-    app:strokeWidth="1dp">
+    android:background="@drawable/movies_bg"
+    android:foreground="@drawable/ripple">
```

Comment:
you can remove this nesting. Just add background to ConstraintLayout and add additional view as overlay.